augly/video/augmenters/ffmpeg/color_jitter.py

```diff
@@ -25,25 +24,28 @@ def __init__(
         self.contrast_level = contrast_level
         self.saturation_level = saturation_level
 
-    def add_augmenter(
-        self, in_stream: FilterableStream, **kwargs
-    ) -> Tuple[FilterableStream, Dict]:
+    def get_command(self, video_path: str, output_path: str) -> List[str]:
         """
         Color jitters the video
 
-        @param in_stream: the FFMPEG object of the video
+        @param video_path: the path to the video to be augmented
 
-        @returns: a tuple containing the FFMPEG object with the augmentation
-            applied and a dictionary with any output arguments as necessary
+        @param output_path: the path in which the resulting video will be stored.
+
+        @returns: a list of strings containing the CLI FFMPEG command for
+            the augmentation
         """
-        return (
-            in_stream.video.filter(
-                "eq",
-                **{
-                    "brightness": self.brightness_level,
-                    "contrast": self.contrast_level,
-                    "saturation": self.saturation_level,
-                }
-            ),
-            {},
-        )
+        command = [
+            "-y",
+            "-i",
+            video_path,
+            "-vf",
+            f"eq=brightness={self.brightness_level}:contrast={self.contrast_level}:saturation={self.saturation_level}",
+            "-c:a",
+            "copy",
+            "-preset",
+            "ultrafast",
+            output_path,
+        ]
```

Comment:
```suggestion
        brightness, contrast, saturation = (
            self.brightness_level, self.contrast_level, self.saturation_level
        )
        command = [
            "-y",
            "-i",
            video_path,
            "-vf",
            f"eq=brightness={brightness}:contrast={contrast}:saturation={saturation}",
            "-c:a",
            "copy",
            "-preset",
            "ultrafast",
            output_path,
        ]
```
Line 43 is >90 lines, let's avoid that :)